source/views/menus/menu-bonapp.js

```diff
@@ -94,19 +95,27 @@ export class BonAppHostedMenu extends React.PureComponent<Props, State> {
 		})
 	}
 
-	requestMenu = (cafeId: string) => () =>
-		fetchJsonQuery(bonappMenuBaseUrl, {cafe: cafeId})
-	requestCafe = (cafeId: string) => () =>
-		fetchJsonQuery(bonappCafeBaseUrl, {cafe: cafeId})
-
 	fetchData = async (props: Props) => {
 		let cafeMenu: ?MenuInfoType = null
 		let cafeInfo: ?CafeInfoType = null
 
+		let menuUrl
+		let cafeUrl
```

Comment:
Can we please assign a default to these variables? `menuUrl` and `cafeUrl`

Reply:
No. They throw an error if they're not assigned, though. 

Well, I suppose we could start with = null; would you like that?

Reply:
Okay, it’s fine as-is.

Reply:
Side channel conversation with Drew has said I can merge without changing this.